app/components/ChapterList.js

```diff
@@ -40,18 +40,21 @@ ChapterList.propTypes = {
 const styles = StyleSheet.create({
   menu: {
     backgroundColor: 'white',
-    paddingTop: 26,
-    paddingLeft: 12,
+    paddingTop: 10,
+    paddingLeft: 20,
     paddingRight: 12,
     paddingBottom: 20,
+    backgroundColor: "#FFF8E2",
   },
   item: {
+    color: "#615334",
```

Comment:
should we add a constants file where all these color strings can live? That way a color like "#615334" could be reused across components and we can update it in one place if the designs change.